scripts-available/CDB_TableUtils.sql

```diff
@@ -88,6 +94,16 @@ BEGIN
       );
     END IF;
 
+    -- Manually record table updates
+    PERFORM public.CDB_TableMetadataTouch(dest_table);
+
+    -- Reenable trigger for tracking updates to after manual run
+    EXECUTE FORMAT(
+      'ALTER TABLE %I.%I ENABLE TRIGGER track_updates',
+      schema_name,
+      dest_table_name
+    );
+
     -- Drop source table
     EXECUTE FORMAT('DROP TABLE %I', source_table_name);
```

Comment:
Tied to the above comment about truncate/insert vs drop/replace logic.  This line needs to also change to the following:

```suggestion
    EXECUTE FORMAT('DROP TABLE %I.%I', schema_name, source_table_name);
```

Reply:
Updated with a version that specifies the schema, along with tests.